serving/server.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+
+import joblib
+from api_models import Properties
+from backend.calculate_level import calculate_level
+from fastapi import FastAPI
+from starlette.middleware.cors import CORSMiddleware
+
+
+app = FastAPI()
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+model = joblib.load(filename="../saved_models/current_model.pkl")
+properties: Optional[Properties] = None
+
+
+@app.get("/", tags=["root"])
```

Comment:
Po co? Jeżeli potrzebujecie jakiegoś "pustego" endpointu, to polecam zrobić healthcheck (`/health`), który jest też dobrą praktyką ogólnie.